.travis.yml

```diff
@@ -9,6 +9,7 @@ addons:
     token: $SONARCLOUD_TOKEN
 script:
   - sonar-scanner
+  - ./gradlew check
   
 notifications:
-  slack: letech:6iQNboxAjhcmAx2B0CW7LpJq
```

Comment:
Travis showed alert concerning this so I put it in env var

Reply:
regenerated the token in slack, added to travis